sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 5.3.1 (2020-10-30)
+### Bug fixes
+- Eagerly close top-level client after fetching the list of partitions instead of waiting until the connection times
```

Comment:
Better mention context like event proccessor load balancer. A user might think the EventHub client getPartitionIds() will close the connection immediately.

Reply:
Good point. Updated the changelog.